src/ProjectTemplates/Web.Spa.ProjectTemplates/Angular-CSharp.njsproj.in

```diff
@@ -0,0 +1,163 @@
+    <Folder Include="e2e\src" />
+    <Folder Include="src\api-authorization" Condition="'$(IndividualLocalAuth)' == 'true'" />
+    <Folder Include="src\app" />
+    <Folder Include="src\assets" />
+    <Folder Include="src\environments" />
+    <Folder Include="src\api-authorization\login" Condition="'$(IndividualLocalAuth)' == 'true'" />
+    <Folder Include="src\api-authorization\login-menu" Condition="'$(IndividualLocalAuth)' == 'true'" />
+    <Folder Include="src\api-authorization\logout" Condition="'$(IndividualLocalAuth)' == 'true'" />
+    <Folder Include="src\app\counter" />
+    <Folder Include="src\app\fetch-data" />
+    <Folder Include="src\app\home" />
+    <Folder Include="src\app\nav-menu" />
+  </ItemGroup>
+  <PropertyGroup>
+    <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);EnsureHttpsDevelopmentCertificate</PrepareForBuildDependsOn>
+    <PostBuildEvent>
+      npm run build
+    </PostBuildEvent>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\Node.js Tools\Microsoft.NodejsToolsV2.targets" />
+  <ProjectExtensions>
+    <VisualStudio>
+      <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
+        <WebProjectProperties>
+          <UseIIS>False</UseIIS>
+          <AutoAssignPort>True</AutoAssignPort>
+          <DevelopmentServerPort>0</DevelopmentServerPort>
+          <DevelopmentServerVPath>/</DevelopmentServerVPath>
+          <IISUrl>http://localhost:48022/</IISUrl>
+          <NTLMAuthentication>False</NTLMAuthentication>
+          <UseCustomServer>True</UseCustomServer>
+          <CustomServerUrl>https://localhost:5002</CustomServerUrl>
+          <SaveServerSettingsInUserFile>False</SaveServerSettingsInUserFile>
+        </WebProjectProperties>
+      </FlavorProperties>
+      <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}" User="">
+        <WebProjectProperties>
+          <StartPageUrl>
+          </StartPageUrl>
+          <StartAction>CurrentPage</StartAction>
+          <AspNetDebugging>True</AspNetDebugging>
+          <SilverlightDebugging>False</SilverlightDebugging>
+          <NativeDebugging>False</NativeDebugging>
+          <SQLDebugging>False</SQLDebugging>
+          <ExternalProgram>
+          </ExternalProgram>
+          <StartExternalURL>
+          </StartExternalURL>
+          <StartCmdLineArguments>
+          </StartCmdLineArguments>
+          <StartWorkingDirectory>
+          </StartWorkingDirectory>
+          <EnableENC>False</EnableENC>
+          <AlwaysStartWebServerOnDebug>False</AlwaysStartWebServerOnDebug>
+        </WebProjectProperties>
+      </FlavorProperties>
+    </VisualStudio>
+  </ProjectExtensions>
+  <Target Name="EnsureHttpsDevelopmentCertificate" Condition="!Exists('$(AspNetHttpsPemKeyPath)') OR !Exists('$(AspNetHttpsCertificate)')">
+    <Exec Command="echo dotnet dev-certs https --export-path $(AspNetHttpsCertificate) --no-password --key-format pem" WorkingDirectory="$(MSBuildProjectDirectory)" />
```

Comment:
This is temporary since I'm waiting on an updated SDK and to merge the changes in https://github.com/dotnet/aspnetcore/pull/23803